tools/report-converter/tests/unit/sparse_output_test_files/Makefile

```
@@ -0,0 +1,4 @@
+all:
+    # Due to difference in format on kernel and command below, and to 
+	# test all the cases of output, the output is different in sample.out file
+    sparse -Wsparse-all sample.c > ./sample.out || exit 0;
```

I tried to run this Makefile but it is failed. The first reason is that sample.c doesn't exist beside the Makefile (it can be found under the files directory), and the indentation is not good. Use the following code:
```suggestion
all:
	# Due to difference in format on kernel and command below, and to 
	# test all the cases of output, the output is different in sample.out file
	sparse -Wsparse-all files/sample.c > ./sample.out 2>&1 || exit 0;

```